core-bundle/src/Resources/config/services.yml

```diff
@@ -181,6 +185,18 @@ services:
         tags:
             - { name: 'contao.escargot_subscriber' }
 
+    Contao\CoreBundle\Cron\Cron:
+        arguments:
+            - '@Contao\CoreBundle\Repository\CronJobRepository'
+            - '@?logger'
+        public: true
+
+    Contao\CoreBundle\Cron\LegacyCron:
+        arguments:
+            - '@contao.framework'
+        tags:
+            - { name: 'terminal42_service_annotation' }
```

Comment:
This should not be necessary as the class implements the `ServiceAnnotationInterface` and this config file has `autoconfigure` enabled.